salt/minion.py

```diff
@@ -954,6 +954,13 @@ def _thread_return(cls, minion_instance, opts, data):
         '''
         # this seems awkward at first, but it's a workaround for Windows
         # multiprocessing communication.
+        if sys.platform.startswith('win') and \
+                opts['multiprocessing'] and \
+                not salt.log.is_logging_configured():
+            # We have to re-init the logging system for Windows
+            salt.log.setup_console_logger(log_level=opts.get('log_level', 'info'))
+            if opts.get('log_file'):
+                salt.log.setup_logfile_logger(opts['log_file'], opts.get('log_level', 'info'))
```

Comment:
@cachedout Shouldn't the second parameter be `log_level_logfile` since it's setting up the log file itself?